app/serializers/api/admin/order_cycle_serializer.rb

```diff
@@ -20,6 +20,10 @@ def viewing_as_coordinator
     Enterprise.managed_by(options[:current_user]).include? object.coordinator
   end
 
+  def subscriptions_count
+    ProxyOrder.not_canceled.where(order_cycle_id: object.id).count
```

Comment:
Second time seeing this code. Maybe we should create `ProxyOrder.count_active(order_cycle)`.

Reply:
Yep, or maybe I should just an a `subscriptions` association to order cycle as per my comment above?